### PR TITLE
Remote Notification Support

### PR DESCRIPTION
**Pull request by helenaford**

### CODE REVIEW

File: docs/react-native/docs/ios/remote-notification-support.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+<Vimeo id="remote-notification-support-1" caption="Step 1 - Add Your Notification Service Extension" />
+
+## Add target to the Podfile
+
+Ensure that your new extension has access to `NotifeeExtensionHelper` by adding it in the Podfile:
+* From the Navigator open the Podfile: Pods > Podfile
+* Scroll down to the bottom of the file and add
+
+```yaml
```
Comment (mikehardy) [inline docs/react-native/docs/ios/remote-notification-support.md:30]: ruby isn't it?

File: docs/react-native/docs/ios/remote-notification-support.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+<Vimeo id="remote-notification-support-2" caption="Step 2 - Update Your Pods" />
+
+## Use the extension helper
+
+At this point everything should still be running normally. This is the final step which is invoking the extension helper.
+
+* From the navigator select your NotifeeNotificationService extension
+* Open the NotificationService.m file
+* At the top of the file import NotifeeExtensionHelper.h right after the NotificationService.h as shown below
+```
```
Comment (mikehardy) [inline docs/react-native/docs/ios/remote-notification-support.md:52]: objective-c I think does markup right (I seem to remember somewhere obj-c wasn't as compatible)

File: docs/react-native/docs/ios/remote-notification-support.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
++ [[NotifeeExtensionHelper extensionHelper] populateNotificationContent:self.bestAttemptContent withContentHandler:contentHandler];
+```
+<Vimeo id="remote-notification-support-3" caption="Step 3 - Edit NotificationService.m" />
+
+Before, moving to the next step, run the app and check it builds successfully – make sure you have the correct target selected. 
+
+## Edit the payload
+Now everything is setup in your app, you can alter your notification payload to include `notifee_options`:
+
+```js
```
Comment (mikehardy) [inline docs/react-native/docs/ios/remote-notification-support.md:71]: json not js?

Comment (helenaford) [reply docs/react-native/docs/ios/remote-notification-support.md:71]: thanks for taking a look 🙌  this was on my todo to double check those types. will update them 😎

File: docs/react-native/docs/ios/remote-notification-support.md (hunk 2 above)
Comment (helenaford) [reply docs/react-native/docs/ios/remote-notification-support.md:52]: used this as a reference https://github.com/github/linguist/blob/master/lib/linguist/languages.yml, went with objectivec

**Pull request merged**